guides/doc-Upgrading_and_Updating/topics/preparing_to_migrate_pulp_content.adoc

```diff
@@ -7,6 +7,14 @@ For large systems, this can mean several days of downtime.
 To prevent this, pre-migrate Pulp content while running the latest version of {ProjectServer} {ProjectVersionPrevious}.
 This reduces the overall upgrade downtime.
 
+During migration to Pulp3, amount of data stored in the `/var/lib/pulp` directory can be doubled.
```

Comment:
/var/lib/pulp shouldn't double, but /var/lib/pulp/published/  would double.  If the user has run satellite-maintain prep-6.10-upgrade, the content in /var/lib/pulp/content should not need to be duplicated. Maybe its a good idea to emphasize that part here?

Reply:
I am not sure if I fully understand how the prep-upgrade is supposed to work. Can you help me to articulate this properly?